org.lflang/src/org/lflang/cli/Lfc.java

```diff
@@ -32,7 +29,14 @@
  *
  * @author {Marten Lohstroh <marten@berkeley.edu>}
  * @author {Christian Menard <christian.menard@tu-dresden.de>}
+ * @author {Atharva Patil <atharva.patil@berkeley.edu>}
  */
+@Command(
+    name = "lfc",
+    // Enable usageHelp (--help) and versionHelp (--version) options.
+    mixinStandardHelpOptions = true,
+    // TODO: Import version from StringsBundle.properties. 
```

Comment:
This `TODO` actually needs fixing before we can merge this because otherwise the version number with be incorrect (as it now already is).

Reply:
Also, shouldn't it just read the version string (without the `lfc ` part?) Not sure though.

Reply:
It should be fixed with this commit: ([2d0662d](https://github.com/lf-lang/lingua-franca/pull/1534/commits/2d0662da45aee7c19e6e32bdc12e587df9e23f35)).

Reply:
Maybe an older version of the PR was shown in the review?

Reply:
Yes, I was looking at an old version. Sorry for the noise! It's unclear to be what `IVersionProvider,getVersion()` is supposed to return though. Should it include the name of the command?

Reply:
True, I just added comments to make it more clear: [4768d34](https://github.com/lf-lang/lingua-franca/pull/1534/commits/4768d3405e94ba9d4bc2bbe6ba4265ecaeaeae30).

Reply:
Sorry, I completely missed your point about whether we want to include the command name in the version info. To clarify, yes, based on previous tests.